legacy/nosql/nosqltest/nosqltest.go

```diff
@@ -382,7 +415,6 @@ func testDeleteByKey(t *testing.T, c tableConf) {
 	})
 
 	del := keys[:5]
-	keys = keys[len(del):]
```

Comment:
Even if this does nothing, please keep it. It shows that both slices must be adjusted.